bosh/secrets.example.yml

```diff
@@ -34,12 +33,40 @@ properties:
     component_name: cf-swarm-containers-broker
     docker_url: secret
 
-
 networks:
 - name: default
-  type: dynamic
+  type: manual
   cloud_properties:
     security_groups:
     - sec_group
     subnet: subnet
+  static:
+  - <changme>
+  dns:
+  - <changeme>
+  gateway: <changeme>
+  reserved:
+  - <changeme>
+  range: <changeme>
```

Comment:
IIRC, bosh will die if we ask for overlapping ranges. To get this working on staging, I picked a range that included the current IPs of the broker and docker machines.